Makefile

```diff
@@ -17,20 +17,34 @@ endif
 VIRSH_CONNECT ?= qemu:///system
 virsh = virsh --connect=$(VIRSH_CONNECT)
 
-CLUSTER_DOMAIN ?= redhat.com
+NET_0_NAME ?= ib-orchestrate-0
+NET_0_BRIDGE_NAME ?= ibo0
+NET_0_DOMAIN ?= ibo0.redhat.com
+NET_0_UUID ?= 268ce8c5-afa5-4076-8c48-963db001bf7d
+NET_0_MAC ?= 52:54:00:1b:1b:00
+NET_0_NETWORK ?= 192.168.126.0/24
+
+NET_1_NAME ?= ib-orchestrate-1
+NET_1_BRIDGE_NAME ?= ibo1
+NET_1_DOMAIN ?= ibo1.redhat.com
+NET_1_UUID ?= 12a4d419-dd03-43c2-8414-ad3339913c02
+NET_1_MAC ?= 52:54:00:1b:1b:01
+NET_1_NETWORK ?= 192.168.127.0/24
```

Comment:
Consider renaming to NET_SEED_... and NET_RECIPIENT_...

Reply:
Also consider moving all these to some network.env file and `include network.env` here

Reply:
Done! Now seed uses NET_SEED and recipient and ibi use NET_RECIPIENT